kamrons-game.py

```diff
@@ -20,137 +28,247 @@ def get_answer(str):
     if loop == 0:
         answer = get_answer("Do you want to play a game? (yes/no)")
         #answer = input("Do you want to play a game? (yes/no)").lower().strip()
-    else:
+        loop = 7
+
+    elif loop == 69:
+        answer = get_answer("Type YES to play and NO to exit")
+        if answer == "no":
+            typewriter("That's too bad - Maybe later!")
+            
+        
+    elif loop == 7:
         answer = get_answer("Do you want to play again? (yes/no)")
-    if answer == "yes":
-        loop += 1
-        answer = get_answer("You are being chased by the FBI for no apparent reason. You run down the sidewalk with the FBI right on your tail, when you reach an intersection. Would you like to go LEFT or RIGHT?")
-        if answer == "left":
+        
+    if answer == "no":
+        typewriter("Okay, maybe next time!")
+
+        break
 
-            answer = get_answer("You run down the sidewalk and reach a dead end. Would you like to CLIMB the wall in front of you or try to foolishly FIGHT the FBI?")
+    elif answer == "yes":
+        loop = 7
+        
+        answer = get_answer("Do you want to get right into the game, or play a short tutorial first? (Type 'game' if you want to play the game, and type 'tutorial' if you want to play a tutorial.)")
         
-            if answer == "climb":
+        if answer == "tutorial":
+            typewriter("The game works like this:")
+            typewriter("Whenever you are given a choice, the command you have to type is in all caps. Got it? Good. Now run the program again and play!")
+            loop = 69
+            sleep(15)
```

Comment:
I recommend having standardized sleep times at the very top of the file.

For example:
```
SUPER_LONG_DELAY = 20
LONG_DELAY = 15
SHORT_DELAY = 10
TINY_DELAY = 5
MICRO_DELAY = 2
```
and then you can sleep for those amounts: `sleep(LONG_DELAY)`.

That way, you can change all of the long delays by just changing the constant.

Reply:
done